usr/sbin/p11sak/p11sak.c

```diff
@@ -9235,7 +9325,10 @@ static CK_RV p11sak_export_sym_key(const struct p11sak_objtype *keytype,
     }
 
 done:
-    free(data);
+    if (data != NULL) {
+        OPENSSL_cleanse(data, data_len);
+        free(data);
```

Comment:
Could you probably use OPENSSL_clear_free instead of the two calls?

Reply:
No. the `data` has been allocated with plan `malloc`, not with `OPENSSL_malloc`, so it must also be freed with plain `free`, and NOT with `OPENSSL_[clear_]free`. You can not mix those!

Reply:
ok, looks good then, will approve.